src/models/BrowserStyleSheet.js

```diff
@@ -191,6 +191,20 @@ if (!DISABLE_SPEEDY) {
       })
     }
 
+    removeComponent(componentId: string) {
```

Comment:
The StyleSheet implementation will change a lot in the future. I'd like to get that refactor PR in before your PR though. Sorry...

I'll make sure to implement the methods you need after it's merged (#1501)

Btw this won't work correctly for the speedy implementation. But no worries I'll make sure that you get the methods you need 🙌

Reply:
Rebuilt the relevant changes on top of your PR, using `StyleSheet.remove` method now